CMakeLists.txt

```diff
@@ -27,13 +27,19 @@ if(NOT CMAKE_INSTALL_PREFIX)
 endif()
 
 project(dawn C CXX)
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.14.5)
```

Comment:
`FetchContent_Populate` was already in CMake 3.11. Version 3.14.5 added `FetchContent_MakeAvailable`. I am fine with requiring 3.14.5 (if this is the version we are going to test), but I want to make sure that this is what you intend to do.